lua/starfall/libs_sh/color.lua

```diff
@@ -5,6 +5,33 @@ local dgetmeta = debug.getmetatable
 local math_Clamp = math.Clamp
 local clamp = function(v) return math_Clamp(v, 0, 255) end
 
+local string_sub = string.sub
```

Comment:
If you're going through the trouble of localizing string.sub, tonumber would be good too I think

Reply:
Actually not necessary if you use the below implementation